Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:12
```

Comment:
if we are using node:12 here then i think we have to change the version of node in the [CI/CD config file](https://github.com/moduslabs/modir/blob/master/.circleci/config.yml#L30)

Reply:
I tried node:14, but the docker-prod-build.sh script failed due to node sass incompatibilities.  I tried uninstalling and reinstalling node sass to no avail.

If it works in Ci/CD config file as is, then I'm not sure why.  I am using MacOS (but Docker == Linux)

Reply:
https://stackoverflow.com/questions/37415134/error-node-sass-does-not-yet-support-your-current-environment-windows-64-bit-w

That kind of error with node > 12

Reply:
oh, oh nice.